Assets/MixedRealityToolkit.Services/InputSystem/GazeProvider.cs

```diff
@@ -163,8 +163,6 @@ public IMixedRealityInputSource GazeInputSource
 
         private Ray latestEyeGaze = default(Ray);
         private DateTime latestEyeTrackingUpdate = DateTime.MinValue;
-        private bool? isEyeCalibrated = null;
-
         private readonly float maxEyeTrackingTimeoutInSeconds = 2.0f;
 
         #region InternalGazePointer Class
```

Comment:
I think that having the InternalGazePointer class be a self-contained entity within the GazeProvider can lead to confusion for developers; especially when there's a UI element in the editor that allows them to define pointers (other than for gaze). 

It doesn't need to be addressed in this PR, but can a work item be opened up to address this concern in the future? While being able to define your own GazeProvider means you also are able to define your own InternalGazePointer; it isn't very easily discoverable.

Reply:
Also, in the OnPreSceneQuery function for the internal gaze pointer; there's this line:
`gazeProvider.HitPosition = Rays[0].Origin + (gazeProvider.lastHitDistance * Rays[0].Direction);
`
Which then gets blown away by the call to UpdateGazeInfoFromHit() from the FocusProvider.

Reply:
Agreed! I'll file an issue about InternalGazePointer

Reply:
https://github.com/microsoft/MixedRealityToolkit-Unity/issues/5612